src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -40,17 +40,27 @@ public CommandResult execute() {
         return new CommandResult(getMessageForPersonListShownSummary(personsFound), personsFound);
     }
 
+    private Set<String> getSetInLowerCase(Set<String> wordSet){
```

Comment:
Can you think of a more appropriate method name? (Hint: `get<What>SetInLowerCase`?)